addons/web/static/src/core/l10n/dates.js

```diff
@@ -164,6 +164,20 @@ export const luxonToMomentFormat = memoize(function luxonToMomentFormat(format)
     });
 });
 
+/**
+ * Maps a luxon locale to the corresponding moment.js locale
+ * 
+ * @param {string} locale luxon identifier of locale to convert
+ * @returns moment identifier of locale
+ */
+export function luxonToMomentLocale(locale) {
```

Comment:
I guess one alternative would be the following

```diff
diff --git a/addons/web/static/src/core/l10n/localization_service.js b/addons/web/static/src/core/l10n/localization_service.js
index 2188cfb05f0c..f11d35a5def7 100644
--- a/addons/web/static/src/core/l10n/localization_service.js
+++ b/addons/web/static/src/core/l10n/localization_service.js
@@ -61,7 +61,9 @@ export const localizationService = {
             // Setup lang inside luxon. The locale codes received from the server contain "_",
             // whereas the Intl codes use "-" (Unicode BCP 47). There's only one exception, which
             // is locale "sr@latin", for which we manually fallback to the "sr-Latn-RS" locale.
-            const locale = lang === "sr@latin" ? "sr-Latn-RS" : lang.replace(/_/g, "-");
+            const locale = lang === "sr@latin" ? "sr-Latn-RS"
+                : lang === "sr_RS" ? "sr-cyrl"
+                : lang.replace(/_/g, "-");
             Settings.defaultLocale = locale;
             for (const [re, numberingSystem] of NUMBERING_SYSTEMS) {
                 if (re.test(locale)) {
```

This changes the default luxon locale from which dates are created.
But maybe we sometimes pass the locale props to the DatePicker, or a date created without the default locale ?
In that case, a function is needed :+1:

Reply:
(Also I couldn't reproduce the crash with the sr@latin locale. Not sure if the locale is correctly used though)